kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/MetricsConstants.scala

```diff
@@ -39,4 +39,10 @@ object MetricsConstants {
   final val ENGINE_FAIL: String = ENGINE + "failed"
   final val ENGINE_TIMEOUT: String = ENGINE + "timeout"
   final val ENGINE_TOTAL: String = ENGINE + "total"
+
+  final private val OPERATION = KYUUBI + "operation."
```

Comment:
The current statement metrics is a subset of operation ones after this PR, will this be redundant?

Reply:
maybe we shall unify them

Reply:
Yes, operation metrics contains statement metrics. I will try to unify them.